public/css/style.css

```diff
@@ -0,0 +1,83 @@
+/* Great source: https://css-tricks.com/hexagons-and-beyond-flexible-responsive-grid-patterns-sans-media-queries/ */
+
+#board {
+    display: grid;
+    place-self: center;
+    justify-content: center;
+    margin: 20px;
+    padding: 20px;
+    background: #333;
+}
+
+.main {
+    display: flex;
+    flex-direction: column;
+    /* size  */
+    --s: 30px;
+    /* margin */
+    --m: 3px;
+    /* height equivalent to 2 overlapping hexagons, used to move every second row */
+    --f: calc(1.732 * var(--s) + 4 * var(--m) - 1px);
+}
+
+.container {
+    /*disable white space between inline block element */
+    font-size: 0;
+    width: calc((var(--nr) + (var(--nr)) / 2)*(var(--s) + 2*var(--m)));
+    padding-bottom: calc(var(--s) * 0.2885 * 2);
+    padding-top: calc(var(--s) * 0.2885 * 2);
+}
+
+.container div {
+    width: var(--s);
+    margin: var(--m);
+    height: calc(var(--s)*1.1547);
+    display: inline-block;
+    font-size: initial;
+    clip-path: polygon(0% 25%, 0% 75%, 50% 100%, 100% 75%, 100% 25%, 50% 0%);
+    margin-bottom: calc(var(--m) - var(--s)*0.2885);
+}
+
+.container::before,
+.container i {
```

Comment:
I'm not sure what this represents, I'm a bit afraid that this will make hover on hexagons fail.